src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/globalmodule.cpp

```diff
@@ -82,15 +87,16 @@ ASPNET_CORE_GLOBAL_MODULE::OnGlobalConfigurationChange(
 
     // Test for an error.
```

Comment:
This comment could be better. Not sure what error it's talking about 😆

Reply:
I think it was referring to the null check.